config/routes.rb

```diff
@@ -4,8 +4,8 @@
   root 'users#index'
  
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show, :new, :create] do
-      resources :comments, only: [:new, :create]
+    resources :posts, only: [:index, :show, :new, :create, :destroy] do
+      resources :comments, only: [:new, :create, :destroy]
     end
   end
```

Comment:
[OPTIONAL]
- [ ] It would be nice to add the likes routes to the `config/routes.rb` file to allow a user to like a post using the UI.

```rb
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, :new, :create, :destroy] do
      resources :comments, only: [:new, :create, :destroy]
      resources :likes, only: [:create]
    end
  end
```